libs/community/langchain_community/tools/azure_cognitive_services/content_safety.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
```

Comment:
Should this be under tools/azure_ai_services instead? My understanding is that AI services is the newer version of the toolkit and replaces the tools in tools/azure_cognitive_services.